tests/integration/test_numpy2.pyi

```diff
@@ -1,11 +1,52 @@
-from typing import Any
+# mypy: disable-error-code="no-redef"
```

Comment:
is this needed for the redefinition of `_`?

Reply:
I didn't want to name all the variables.